docs/kpod-push.1.md

```diff
@@ -55,9 +55,13 @@ Credentials (USERNAME:PASSWORD) to use for authenticating to a registry
 
 Pathname of a directory containing TLS certificates and keys
 
-**--disable-compression, -D**
+**--compress**
 
-Don't compress copies of filesystem layers which will be pushed
+Compress tarball image layers when saving to directory using the 'dir' transport. (default is same compression type as source)
```

Comment:
are there other compression types that can be listed?  Can you add at least one example that's not using the default compression type?

Reply:
Its just a bool. Added the option of being uncompressed or compressed to the statement.